src/components/layout/layout.js

```diff
@@ -133,22 +135,21 @@ const Layout = ({
                 (!is_eu_country || tracking_status === 'accepted') && !gtm_data && has_dataLayer
 
             if (allow_tracking) {
-                setGTMData({ event: 'allow_tracking' })
+                setGTMData({
+                    event: 'allow_tracking',
+                    LoggedIn: is_logged_in,
```

Comment:
this should be set by default, so we dont need to pass this everytime, we can move these two to the hooks, and make our own custom setters

Reply:
```suggestion
                    loggedIn: is_logged_in,
```